scripts/corrigir-situacao-2026-stale.py

```
@@ -0,0 +1,182 @@
+
+
+def main() -> None:
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--apply", action="store_true")
+    args = ap.parse_args()
+
+    lista = ",".join(SLUGS)
+    st, fichas = rest(f"candidatos?slug=in.({lista})"
+                      f"&select=id,slug,nome_urna,estado,situacao_candidatura,sq_candidato_2026")
+    if st != 200 or not fichas:
+        raise SystemExit(f"ABORTA: leitura das fichas falhou ({st})")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)